app/views/application/_page_footer.html.haml

```diff
@@ -11,3 +11,6 @@
         = fa_icon 'linkedin'
       %a.share-mail{href: "mailto:?subject=Sveriges%20Hundf%C3%B6retagare&body=#{request.url}"}
         = fa_icon 'envelope'
+    .geo-ssl-seal
+      %a.ssl-seal{ href: "https://www.rapidssl.com/about/shareArticle?mini=true&title=Sveriges%20Hundf%C3%B6retagare&url=#{request.url}", target: '_blank'}
+        = image_tag('RapidSSL_SEAL-90x50.gif')
```

Comment:
1. This adds a lot of white space at the end of the page:

<img width="356" alt="Screen Shot 2019-03-09 at 9 17 57 AM" src="https://user-images.githubusercontent.com/9968213/54072726-c44f7480-424c-11e9-8606-0c1ddc44606e.png">

What about positioning the image in left-hand or right-hand corner, in-line with the social media icons?  (or some other less-intrusive place)?  @thesuss any opinions on this?

2. When I click on the icon I see this:

<img width="753" alt="Screen Shot 2019-03-09 at 9 24 47 AM" src="https://user-images.githubusercontent.com/9968213/54072747-345dfa80-424d-11e9-95ff-dcaf0fdcbc7e.png">

3. Can we get this image in Swedish?  (if not, no big deal, but it would be nice if we could).  (and, if so, we should switch images as we switch language).

Reply:
I think it should be on the pages with payments. I E company page and my account when logged in. 
The placement should be top right or bottom of the actual page content of those pages, not in the footer. 
Then it will work well in the redesign also.

Reply:
FWIW - As with Patrick, when I click on the RapidSSL image I get the error page.  What is it supposed to go to?  (And can you write a step for that so it's tested in a scenario?  "When I click on <the image> I see <content expected in the RapidSSL page....>" or something like that.

Reply:
Yes it is not my proudest work. To be honest the image itself is terrible, but then again, it is what they have chosen and you cannot really change it because then it's not the genuine badge any longer. 

I agree it could be placed in specific pages, which was my first inital thought, but I need some help thinking of how to do that. 

Should you place it in the footer partial as now or create a specific partial that adds to the current one? 
Should there be a condition and do you place that condition in the view? Or elsewhere?
How do you know which page is calling the partial in order to place a condition that works?

Reply:
I'll get on the 404 right away.

Reply:
shouldn't it be possible to do 
`%img{ src: 'RapidSSL_SEAL-90x50.gif'}` 
instead of the `image_tag`? I tried and failed. Should maybe try again.

Reply:
Better to use the `tag` helpers because then Rails will handle so much of all of the asset pipeline stuff for you.  

FYI:  Need to have text in the `alt` tag.  This is used by screen-readers (think _**alt**ernate rendering devices_) and in SEO scores.  
The `title` tag is usually used to create a tooltip (and/or hover) box by browsers.  Less important.

But good practice is to make sure you have good info in the `alt` tag and then have the default value of the `title` tag = what you have for the `alt` tag.

So, working from Patrick's example, I'd update it to this:
```
.geo-ssl-seal
  = link_to(image_tag('RapidSSL_SEAL-90x50.gif', alt: 'Secured by RapidSSL',
            title: 'RapidSSL seal'),
            'https://www.rapidssl.com/about/',
            target: '_blank', rel: 'nofollow')
```
For the alt tag I used the text that RapidSSL uses on their site.

Reply:
@patmbolger What is canonical Rails? Does it have to do with canonical links for SEO? I Googled it and found an interesting gem https://blog.planetargon.com/entries/easily-add-canonical-urls-to-your-rails-app

Reply:
@weedySeaDragon Thank you for tips, I actually am quite anal with img attribs like that, but this time I just wanted to get it working. Will re-direct my mindset to be meticulous from the start. :)

Reply:
@thesuss I tried to log in on my localhost with "arne@random.com" but it didn't work. How can I see the layout of the pages you mentioned? Oh, so rusty.

Reply:
@rmzse I'm actually using "canonical" in the sense of being the standard, accepted and/or preferred way to do things.  In other words, "The Rails Way".  (which reminds me of the excellent "rails way" series of books - now up to 5 - https://www.amazon.com/Rails-Way-Addison-Wesley-Professional-Ruby/dp/0134657675 - more a reference book than a guide, but worth having).

Anyway, something that is "canonical" could be seen as being too restrictive, but I think it works for rails because rails is nothing if not opinionated, and rails _loves_ constraints and guard rails.

Geez, ask a simple question ...

Reply:
👍  Looking good!  Link is working.  title text is good.

Regarding where it should go (just on the payment page(s)?) etc., we can leave that up to @thesuss .  Perhaps we can go with this and then adjust the location/layout after #578  is finished.

Good job! :-)